app/Http/Controllers/AdministrationController.php

```diff
@@ -44,29 +95,84 @@ public function nominationsSearch(Request $request)
                 $data,
                 function ($var) use ($search, $data) {
                     return AdministrationController::array_any(
-                            $data,
-                            function ($alias) use ($search) {
+                        $data,
+                        function ($alias) use ($search) {
                                 return str_contains(strtolower($alias['nominee_first_name']), strtolower($search));
                             }
-                        ) ||
+                    ) ||
                         AdministrationController::array_any(
                             $data,
                             function ($alias) use ($search) {
-                                return str_contains(strtolower($alias['nominee_last_name']), strtolower($search));
-                            }
+                                    return str_contains(strtolower($alias['nominee_last_name']), strtolower($search));
+                                }
                         ) ||
                         AdministrationController::array_any(
                             $data,
                             function ($alias) use ($search) {
-                                return str_contains(strtolower($alias['nominee_email']), strtolower($search));
-                            }
+                                    return str_contains(strtolower($alias['nominee_email']), strtolower($search));
+                                }
                         );
                 }
             );
             return view('administration/nominations', ["nominations" => $data, "search" => $search]);
         }
     }
 
+    private static function getNomineesData()
+    {
+        $nominees = Nominee::all();
+        if (count($nominees) === 0) {
+            self::FillNomineesTable();
+        }
+        $faculties = Faculty::all();
+        $data = [];
+
+        foreach ($nominees as $nominee) {
+            $nominee['faculty'] = $faculties->find($nominee['faculty_id']);
+            $nominee['achievements'] = implode('; ', unserialize($nominee['achievements']));
+            array_push($data, $nominee);
+        }
+        return $data;
+    }
+
+    private static function FillNomineesTable()
+    {
+        $allowedCategories = [1, 2, 3, 4, 5, 6];
```

Comment:
Notyvole.... vytahni si to z DB to je source of truth ne tady nejaky random pole :D

Reply:
Nehlede na to ze tady zmenis order nejak a nebude to == dbs

Reply:
Done